src/utils/xml_helpers.py

```
@@ -3,30 +3,35 @@
-def create_opml_tree(feeds_by_genre: Dict[str, List[Feed]], title: str = "RSS Feeds") -> ET.Element:
+
+def create_opml_tree(
+    feeds_by_genre: Dict[str, List[Feed]], title: str = "RSS Feeds"
+) -> ET.Element:
     """Create an OPML XML tree from feeds grouped by genre."""
     root = ET.Element("opml", version="1.0")
     head = ET.SubElement(root, "head")
     ET.SubElement(head, "title").text = title
-    ET.SubElement(head, "dateModified").text = datetime.now().strftime("%a, %d %b %Y %H:%M:%S %z")
-    
+    ET.SubElement(head, "dateModified").text = datetime.now().strftime(
```

Ruff (0.8.0)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)